docs/quickstart.rst

```diff
@@ -21,66 +22,89 @@ Installation
             #...,
-            "formatters": {
-                #...,
-                "outgoing_requests": {"()": HttpFormatter},
+            "log_outgoing_requests": {
+                "level": "DEBUG",
+                "formatter": "outgoing_requests",
+                "class": "logging.StreamHandler",  # to write to stdout
             },
-            "handlers": {
-                #...,
-                "log_outgoing_requests": {
-                    "level": "DEBUG",
-                    "formatter": "outgoing_requests",
-                    "class": "logging.StreamHandler",
-                },
-                "save_outgoing_requests": {
-                    "level": "DEBUG",
-                    "class": "log_outgoing_requests.handlers.DatabaseOutgoingRequestsHandler",
-                },
+            "save_outgoing_requests": {
+                "level": "DEBUG",
+                # enabling saving to database
+                "class": "log_outgoing_requests.handlers.DatabaseOutgoingRequestsHandler",
             },
-            "loggers": {
-                #...,
-                "requests": {
-                    "handlers": ["log_outgoing_requests", "save_outgoing_requests"],
-                    "level": "DEBUG",
-                    "propagate": True,
-                },
+        },
+        "loggers": {
+            #...,
+            "requests": {  # the logger name must be 'requests'
+                "handlers": ["log_outgoing_requests", "save_outgoing_requests"],
+                "level": "DEBUG",
+                "propagate": True,
             },
-        }
+        },
+    }
 
-        LOG_OUTGOING_REQUESTS_DB_SAVE = True # save logs enabled/disabled based on the boolean value
-        LOG_OUTGOING_REQUESTS_DB_SAVE_BODY = True # save request/response body
-        LOG_OUTGOING_REQUESTS_EMIT_BODY = True # log request/response body
-        LOG_OUTGOING_REQUESTS_CONTENT_TYPES = [
-                "text/*",
-                "application/json",
-                "application/xml",
-                "application/soap+xml",
-        ] # save request/response bodies with matching content type
-        LOG_OUTGOING_REQUESTS_MAX_CONTENT_LENGTH = 524_288  # maximal size (in bytes) for the request/response body
-        LOG_OUTGOING_REQUESTS_LOG_BODY_TO_STDOUT = True
 
+The library ships with safe defaults for settings - essentially only emitting
+meta-information about requests and responses. To view request and response bodies,
+you likely want to apply the following non-default settings:
 
-#.  Run ``python manage.py migrate`` to create the necessary database tables.
+.. code-block:: python
+
+    LOG_OUTGOING_REQUESTS_DB_SAVE = True # save logs enabled/disabled based on the boolean value
+    LOG_OUTGOING_REQUESTS_DB_SAVE_BODY = True # save request/response body
+    LOG_OUTGOING_REQUESTS_EMIT_BODY = True # log request/response body
+
+.. note:: Saving to database (both the entire record and the request/response bodies)
```

Comment:
Should we mention the possibility of specifying max size here? 

In addition, you can specify the maximum size (in bytes) for contents to be saved, like so:

LOG_OUTGOING_REQUESTS_MAX_CONTENT_LENGTH = 524_288 # 0.5 MB

Reply:
I'm deliberately leaving out the setting here as I believe the default of 0.5 MB should just work. The other remarks are now incorporated!